pkg/watches/watches.go

```diff
@@ -21,20 +21,23 @@ import (
 	"os"
 	"time"
 
-	"gopkg.in/yaml.v2"
+	"gopkg.in/yaml.v2" // todo: remplace for the k8s lib "sigs.k8s.io/yaml"
```

Comment:
I think there was a reason I did this, but I can't remember what it is now. So yeah, we can try to use `sigs.k8s.io/yaml`.

Reply:
I think that is because the Watch struct was done here with using yaml instead of JSON. Kept it until we are able to change.

Reply:
track task for that.